pkgs/build-support/rust/default-crate-overrides.nix

```diff
@@ -270,6 +302,11 @@ in
     nativeBuildInputs = [ protobuf ];
   };
 
+  pyo3 = attrs: {
+    nativeBuildInputs = [ pkg-config ];
+    extraLinkFlags = [ "-L${python3}/lib" "-l${python3.libPrefix}" ];
```

Comment:
It needs pkg-config, but then also needs you to manually specify the link flags rather than getting them from pkg-config?

Reply:
This is because different versions and feature sets complain about different things. This does work though. PyO3 is a bit special, I'd like to also add a variant of `buildRustCrate` specifically to produce Python wheel from dependents of PyO3.

Reply:
If it changed in some version, might be clearer to handle the before/after separately.